administrator/language/en-GB/com_guidedtours.ini

```
@@ -88,4 +94,7 @@ COM_GUIDEDTOURS_TYPE_REDIRECT_URL_DESC="Enter the relative URL of the page you w
 COM_GUIDEDTOURS_TYPE_REDIRECT_URL_LABEL="Relative URL"
 COM_GUIDEDTOURS_URL_LABEL="Relative URL"
 COM_GUIDEDTOURS_URL_DESC="Enter the relative URL of the page from where you want to Start the tour, e.g administrator/index.php?option=com_guidedtours&view=tours for the tours' list page."
+COM_GUIDEDTOURS_USERSTATE_CONNECTEDONLY="Tour User state action is only for connected users."
+COM_GUIDEDTOURS_USERSTATE_STATENOTSAVED="User state not saved for user %s tour %s."
+COM_GUIDEDTOURS_USERSTATE_STATESAVED="User state saved for user %s tour %s."
```

```suggestion
COM_GUIDEDTOURS_USERSTATE_STATESAVED="User state saved for user %1$s tour %2$s."
```


https://docs.joomla.org/Formatted_fields_in_language_translation_strings

**Format Argument Swapping**
The format string supports argument numbering and even swapping. This is useful where two or more data items must be embedded in a string but differences in language structure means that the order of use of the data items is not the same.